app/assets/javascripts/oxalis/model/binary/bucket.js

```diff
@@ -198,10 +202,22 @@ export class DataBucket {
     }
     this.isDownSampled = true;
 
-    const halfBucketWidth = constants.BUCKET_WIDTH / 2;
-    const xOffset = (bucket.zoomedAddress[0] % 2) * halfBucketWidth;
-    const yOffset = (bucket.zoomedAddress[1] % 2) * halfBucketWidth;
-    const zOffset = (bucket.zoomedAddress[2] % 2) * halfBucketWidth;
+    const [xFactor, yFactor, zFactor] = resolutionsFactors;
+
+    if (xFactor !== 2 || yFactor !== 2 || [1, 2].indexOf(zFactor) === -1) {
+      // downsampling for anisotropic datasets is only supported when x and y change
+      // isotropically and z stays the same or doubles its resolution
+      throw new Error("Downsampling initiated for unsupported resolution change");
```

Comment:
Is this visible to user's in any way. (Toast, etc?)

BTW, would you mind updating the Readme to include the supported data formats / limitations for anisotropy etc.      Thanks.

Reply:
> Is this visible to user's in any way. (Toast, etc?)

No, I don't think so. However, this is quite edge-casy. Once there is a datset with such properties and user's zoom out, they will notice that there are no visible buckets anymore. They will raise a bug report and we can see the error then. Not sure whether this will happen anytime soon, though.

> BTW, would you mind updating the Readme to include the supported data formats / limitations for anisotropy etc. Thanks.

Sure, I'll do that.